dali/operators/image/resize/resampling_attr.cc

```diff
@@ -103,8 +155,8 @@ void ResamplingFilterAttr::GetResamplingParams(
       if (resz_par.src_lo[d] != resz_par.src_hi[d])
         rsmp_par.roi = { resz_par.src_lo[d], resz_par.src_hi[d] };
       rsmp_par.output_size = resz_par.dst_size[d];
-      rsmp_par.min_filter = { min_filter_[i], 0 };
-      rsmp_par.mag_filter = { mag_filter_[i], 0 };
+      rsmp_par.min_filter = { min_filter_[i], 0, antialias_};
```

Comment:
antialias only applies to downscaling

Reply:
I think at this point we should also demote `triangular` to `linear`.